test/system/alpha/banner_test.rb

```diff
@@ -21,5 +21,27 @@ def test_dismiss_button_hide
 
       assert_selector(".Banner", visible: :hidden)
     end
+
+    def test_announce_on_show
+      visit_preview(:announce_on_show)
+
+      assert_equal page.evaluate_script("document.querySelector('live-region').shadowRoot.querySelector('[aria-live=\"polite\"]').innerText"), ""
+
+      find("#show-banner").click
+
+      assert_equal page.evaluate_script("document.querySelector('live-region').shadowRoot.querySelector('[aria-live=\"polite\"]').innerText.trim()"), "Hello."
+    end
+
+    def test_announce_on_show_when_hidden_wrapper_is_shown
+      visit_preview(:announce_on_show_with_hidden_wrapper)
+
+      assert_selector("#wrapper", visible: :hidden)
+      assert_equal page.evaluate_script("document.querySelector('live-region').shadowRoot.querySelector('[aria-live=\"polite\"]').innerText"), ""
+
+      find("#toggle-wrapper").click
+
+      assert_selector("#wrapper", visible: :visible)
+      assert_equal page.evaluate_script("document.querySelector('live-region').shadowRoot.querySelector('[aria-live=\"polite\"]').innerText.trim()"), "Hello."
+    end
```

Comment:
I want to add a test for when `Primer::Alpha::Banner` is dynamically injected. How should I set this up?